<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOrderDialog</class>
 <widget class="QDialog" name="CreateOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateOrderDialog</string>
  </property>
  <widget class="QGroupBox" name="positionsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>831</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Позиции заказа</string>
   </property>
   <widget class="QTableWidget" name="positionTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>811</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Описание</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Выдача</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>0</y>
     <width>831</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="4">
     <widget class="QLabel" name="designerLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дизайнер</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="amountLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сумма</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="paymentLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Оплата</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="managersComboBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="paymentComboBox">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дата</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="designersComboBox"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="amountLineEdit">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="managerLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Менеджер</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openCalendarBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>248</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="addPositionBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Добавить позицию</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delPositionBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Удалить позицию</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="quantityGroupBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Количество</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="quantityLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>831</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="addOrderBtn">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Внести заказ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Сохранить изменения</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="descriptionGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Описание позиции заказа</string>
   </property>
   <widget class="QTextEdit" name="descriptionTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>541</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="remarkGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>831</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Примечание к заказу</string>
   </property>
   <widget class="QTextEdit" name="remarkTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>811</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="issueGroupBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Выдача</string>
   </property>
   <widget class="QComboBox" name="issueComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>831</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Клиент</string>
   </property>
   <widget class="QComboBox" name="clientsComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="maxVisibleItems">
     <number>15</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderDialog</receiver>
   <slot>saveChangesSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>672</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openCalendarBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderDialog</receiver>
   <slot>openCalendarSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delPositionBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderDialog</receiver>
   <slot>removePositionSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>916</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPositionBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderDialog</receiver>
   <slot>addPositionSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOrderBtn</sender>
   <signal>clicked()</signal>
   <receiver>CreateOrderDialog</receiver>
   <slot>addOrderToDBSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>675</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>688</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveChangesSlot()</slot>
  <slot>openCalendarSlot()</slot>
  <slot>removePositionSlot()</slot>
  <slot>addPositionSlot()</slot>
  <slot>addOrderToDBSlot()</slot>
  <slot>slot5()</slot>
 </slots>
</ui>
